config/default/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ namePrefix: byoh-
 
 # Labels to add to all resources and selectors.
 #commonLabels:
-#  someName: someValue
+cluster.x-k8s.io/provider: "infrastructure-byoh"
```

Comment:
We need to uncomment commonLabels and it should be like below
```
commonLabels:
  cluster.x-k8s.io/provider: "infrastructure-byoh"
```

Reply:
Okay👍🏽